chromebook-setup.sh

```diff
@@ -704,6 +841,7 @@ ensure_command mkfs.ext4 e2fsprogs
 ensure_command mkimage u-boot-tools
 ensure_command udisksctl udisks2
 ensure_command vbutil_kernel vboot-utils
+ensure_command cgpt vboot-utils
```

Comment:
vboot-utils is checked just in the above line, and cgpt is part of vboot-utils. So this line is not needed.